### DISCUSSION TITLE
May be I fixed a bug?

### DISCUSSION THREAD

**Discussion by bitsarat** [pointfreeco/swift-composable-architecture#2742] [Q&A] [answered]
Hey guys, Congrats on a huge milestone. The new changes make TCA far more approachable. I made a minor fix to the ToDo app, to make sure as yet Untitled Todos cannot be marked complete. Don't know if it is important to you guys, I can submit a PR if needed
<img width="1710" alt="Screenshot 2024-01-29 at 10 59 43 PM" src="https://github.com/pointfreeco/swift-composable-architecture/assets/16160755/f763832c-08e8-44d4-92fd-c3a62eaf689f">
.

**Comment by mbrandonw** [comment, marked as answer]
Hi @bitsarat, we don't consider this a bug in the app. We're only trying to show the most basic usage of a list in TCA.

**Reply by bitsarat** [reply]
Thanks @mbrandonw, agreed. I'm taking baby steps modifying the code as I see fit.
